src/server/common/model/answer/biosecurity-map/biosecurity-map.js

```diff
@@ -0,0 +1,105 @@
+import { AnswerModel } from '../answer-model.js'
+import { validateAnswerAgainstSchema } from '../validation.js'
+import validationSchema from './validation.js'
+
+/**
+ * export @typedef {{
+ *  metadata: {
+ *   uploadId: string;
+ *   uploadUrl: string;
+ *   statusUrl: string;
+ *  }
+ * status?: {
+ *   uploadStatus: 'initiated' | 'pending' | 'ready',
+ *   metadata: {},
+ *   form: {
+ *     crumb: string,
+ *     file: any
+ *   },
+ *   numberOfRejectedFiles: number
+ * }
+ * } | undefined} BiosecurityMapData
+ *
+ * export @typedef {{ metadata: {
```

Comment:
I think you can use BiosecurityMapData here

Reply:
done